src/pages/Follow/Component/Feed.js

```diff
@@ -0,0 +1,227 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { BsHeart, BsBookmark, BsChat, BsShare } from 'react-icons/bs';
+import { FiMoreHorizontal } from 'react-icons/fi';
+import { useNavigate } from 'react-router';
+
+export const Feed = ({ item }) => {
+  const navigate = useNavigate();
+  const [expand, setExpand] = useState(false);
+
+  const handleExpand = () => {
+    const copy = expand;
+    setExpand(!copy);
+  };
+
+  return (
+    <UserFeed>
+      <UserId>
+        <IdWrap>
+          <ProfileImg src={item.writerImg} alt="" />
+          <UserIdText>{item.writer}</UserIdText>
+          <UserWriteDate>1일전</UserWriteDate>
+        </IdWrap>
+        <BtnWrap>
+          <MoreBtn>
+            <FiMoreHorizontal />
+          </MoreBtn>
+        </BtnWrap>
+      </UserId>
+      <UserImgWrap>
+        <UserImg
+          src={item.img}
+          alt=""
+          onClick={item => {
+            navigate(`/card_collections/${item.id}`);
+          }}
+        />
+      </UserImgWrap>
+      <UserTextWrap>
+        <UserText expand={expand}>{item.content}</UserText>
+        {item.content.length > 70 && (
+          <ExpandBtn
+            onClick={() => {
+              handleExpand();
+            }}
```

Comment:
함수를 실행하는 함수를 호출하고 있기 때문에, 해당 함수 자체를 호출하면 됩니다.
```suggestion
            onClick={handleExpand}
```

Reply:
수정하였습니다.